src/Uno.UI/ts/Windows/Storage/Pickers/FileOpenPicker.ts

```diff
@@ -0,0 +1,89 @@
+﻿namespace Windows.Storage.Pickers {
+
+	export class FileOpenPicker {
+		public static isNativeSupported(): boolean {
+			return typeof showOpenFilePicker === "function";
+		}
+
+		public static async nativePickFilesAsync(
+			multiple: boolean,
+			showAllEntry: boolean,
+			fileTypesJson: string): Promise<string> {
+
+			if (!FileOpenPicker.isNativeSupported()) {
+				return JSON.stringify([]);
```

Comment:
`return "[]";` should be enough here ;-)

Reply:
Will fix as part of an upcoming PR :-)